packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/add/vsphere-user-add.controller.js

```diff
@@ -29,18 +29,23 @@ export default class {
   }
 
   addUser() {
+    this.trackClick('add::confirm');
     this.loaders.add = true;
     this.DedicatedCloud.addUser(this.productId, this.newUser).then(
       () => {
-        this.goBack(this.$translate.instant('dedicatedCloud_users_add_start'));
+        return this.goBackWithTrackingPage({
+          message: this.$translate.instant('dedicatedCloud_users_add_start'),
+          trackingTag: 'add-success',
+        });
       },
       (err) => {
-        this.goBack(
-          `${this.$translate.instant(
+        return this.goBackWithTrackingPage({
+          message: `${this.$translate.instant(
             'dedicatedCloud_users_add_error',
           )} ${err.message || err}`,
-          'danger',
-        );
+          type: 'danger',
+          trackingTag: 'add-error',
+        });
```

Comment:
Same as https://github.com/ovh/manager/pull/8158/files#r989753330

Reply:
It's applied.